index.html

```diff
@@ -77,6 +77,45 @@ <h2>
     </section>
 
     
+    <section class="contact_section" id="contact">
+        <h2 class="contact_header">
+          I'm always interested in hearing about new projects, so if you'd like to
```

Comment:
- [ ] Please, consider that you should have a whole page with all sections. You have only toolbar, headline, and contact form sections. Please, add also the "My recent projects" and "About me" sections.
![2022-01-12 21 36 07](https://user-images.githubusercontent.com/49064106/149192523-f0c694b3-6739-4e33-ab33-2d46f6f1b3a8.gif)